### ISSUE TITLE
1 Untracked File

### ISSUE THREAD

**Issue by sebestenyb** [ExpandedFronts/Revisr#78]
Hi

Revisr shows 1 untracked files in the header, but doesn't show anything to commit in the commit view.
Also checking via SSH:

```
$ git status
# On branch master
nothing to commit (working directory clean)
```

Any idea?

Cheers, Sebi

![screen shot 2015-10-04 at 9 57 05 am](https://cloud.githubusercontent.com/assets/743212/10265094/9e5ca936-6a7e-11e5-9ea6-8628e6b7b6a2.png)

**Comment by mgshaw** [comment]
Hi,

Can you check the status via the "Help" tab in the settings page and see if it is the same?

**Comment by sebestenyb** [comment]
Oh cool, I didn't realised that there are raw status like that.
Anyway, here's what happened:

I found out that on the dashboard the logs indicated that Revisr couldn't pull from the repo either, and there it showed me the error, when I retried:

```
fatal: Out of memory? mmap failed: Cannot allocate memory 
fatal: The remote end hung up unexpectedly
```

So I did a `git cg`, what solved it, but to make sure (it was a fresh project) I re-created the repo, and excluded the uploads folder. I reckon there must have been a huge image there freaking out the shared hosting.

Thanks, I think we can close this.
